README.md

```diff
@@ -0,0 +1,40 @@
+# Excalidraw
+
+Excalidraw is a whiteboard tool that lets you easily sketch diagrams.
+
+## Try it now
+
+If you want to start sketching, please check https://excalidraw.com
+
+### Excalidraw is built using these awesome tools
```

Comment:
Can you move this after Getting Started. I don't think it's worth being super high